packages/@ember/-internals/glimmer/lib/helpers/fn.ts

```diff
@@ -6,6 +6,52 @@ import { InternalHelperReference } from '../utils/references';
 import buildUntouchableThis from '../utils/untouchable-this';
 
 const context = buildUntouchableThis('`fn` helper');
+
+/**
+@module ember
+*/
+
+/**
+  The `fn` helper is used to wrap the provided callback function up with a
+  set of arguments. When the returned function is invoked the callback will
+  receive the arguments from the template along with the arguments passed when
+  invoking the returned function. Importantly, the `fn` helper does **not**
+  attempt to ensure any specific `this` context is present within the callback
+  when it is executed. This means that if your callback function needs access to
+  a specific `this` context, you should ensure that it is bound prior to the `fn`
+  invocation.
+
+  For example, given the following component:
+
+  ```app/components/calculator.js
```

Comment:
Does this work? I've only seen examples setting the language

Reply:
I think it does, and there are some other examples in the codebase already.

@mansona / @toddjordan / @jenweber - Can y'all confirm?

Reply:
I don't think this works but I will have a look.

The Guides uses the following syntax but I don't think it's the same for API docs:
```javascript {data-filename=some/file.js}

Reply:
TIL, it does work as @rwjblue has written:

![Screen Shot 2019-04-24 at 5 17 39 PM](https://user-images.githubusercontent.com/16627268/56694562-0164ae80-66b5-11e9-93e1-6c998a416184.png)

![Screen Shot 2019-04-24 at 5 17 51 PM](https://user-images.githubusercontent.com/16627268/56694570-0590cc00-66b5-11e9-96b8-3dbffc98d734.png)